### PR TITLE
Prepare release 2.1.0 alpha.2

### LINKED ISSUE #2621: Prepare 2.1.0-alpha.2 release
- [x] Merge release blockers
    - [x] #2533
    - [x] #2614
- [x] Update changelog
- [x] Bump API version number
- [x] Update README's with API diff links
- [x] Bump backend version number
- [x] Bump required SDK version number
- [x] Release new SDK version
- [x] Check translations
    - [x] Javascript
    - [x] Formio
    - [x] Python

### PR DESCRIPTION
**Pull request by sergei-maertens**
Closes #2621

### CODE REVIEW

File: src/openapi.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.3
 info:
   title: Open Forms API
-  version: 2.0.0
+  version: 2.1.0-beta.0
```
Comment (joeribekker) [inline src/openapi.yaml:4]:
```suggestion
  version: 2.1.0-alpha.2
```

I think the API version should always be the same as the software, even if we think the API will not get any more features and the software will.

File: src/openforms/conf/base.py [hunk 2]
```diff
@@ -832,7 +832,7 @@
 and it plays nice with other available components.
 """
 
-API_VERSION = "2.0.0"
+API_VERSION = "2.1.0-beta.0"
```
Comment (joeribekker) [inline src/openforms/conf/base.py:835]:
```suggestion
API_VERSION = "2.1.0-alpha.2"
```

File: src/openforms/js/components/form/edit/validationEditFormUtils.js [hunk 3]
```diff
@@ -50,19 +50,6 @@ const getCustomValidationErrorMessagesEditForm = validators => {
     collapsible: true,
     collapsed: true,
     components: [
-      {
```
Comment (joeribekker) [inline src/openforms/js/components/form/edit/validationEditFormUtils.js:53]: 😿

File: src/openforms/js/lang/nl.json [hunk 4]
```diff
@@ -1400,7 +1400,7 @@
     "originalDefault": "String"
   },
   "oYmpN5": {
-    "defaultMessage": "Whether to include the content of the confirmation page in the PDF.",
+    "defaultMessage": "Vink aan om de inhoud toe te voegen aan de PDF met gegevens ter bevestiging.",
```
Comment (joeribekker) [inline src/openforms/js/lang/nl.json:1403]:
```suggestion
    "defaultMessage": "Vink aan om de inhoud toe te voegen aan de PDF die mensen op het eind kunnen downloaden."
```

File: src/openapi.yaml (hunk 1 above)
Comment (sergei-maertens) [reply src/openapi.yaml:4]: missed this one, it should be alpha in this case. However, the backend version itself can be newer than the API version if the changes do not affect the API spec at all - e.g. API version 2.1.0 and backend version 2.1.5.

We are documenting the version of the API spec, not the version of the API per the OAS specification:

> REQUIRED. The version of the OpenAPI **document** (which is distinct from the [OpenAPI Specification version](https://swagger.io/specification/#oas-version) or the API implementation version).

(emphasis mine, see https://swagger.io/specification/#info-object)

File: src/openforms/js/components/form/edit/validationEditFormUtils.js (hunk 3 above)
Comment (sergei-maertens) [reply src/openforms/js/components/form/edit/validationEditFormUtils.js:53]: tears were shed indeed, but not because it's been removed

**Pull request merged**